### ISSUE TITLE
Gfycat/webm issue (NOT fixed by 0.3.3.2)

### ISSUE THREAD

**Issue by smit3119** [ejci/Chrome-Audio-EQ#61]
The webm issue is still occurring after updating to 0.3.3.2.  Opening webm links still results in a still image, disabling the extension fixes the issue.

**Comment by thomaskoji** [comment]
Devs have to let the users know about this issue. I was dying for week.

**Comment by ejci** [comment]
Will fix it during the weekend.

**Comment by smit3119** [comment]
ejci, do you happen to have any updates on this?  If you've run into trouble with the fix it's no problem, but some communication would be greatly appreciated.  My music sounds like crap now!

**Comment by katastic** [comment]
YES. I finally found THIS extension was breaking it after months of wondering why my Reddit was broken.

**Comment by smit3119** [comment]
In future, troubleshoot by opening an icognito window (or otherwise turning off all extensions) and see if the problem persists.  If not, then an extension's breaking it.  Turn off all of them and re-enable one at a time until it breaks again.  The bad news is is ejci seems to have either abandoned this extension or been pulled away from it, because the problem's been here for months and there's been very little communication from the dev about when or even if it will be fixed.  I use it to make Google Music sound better, and what I do is enable it, open Google Music, play my stuff, then disable it.  It will continue to be active in that tab until the tab closes, but won't affect other tabs.  Hopefully this helps you too.

**Comment by katastic** [comment]
I use it for YouTube because my subwoofer on my speakers (even on
lowest setting) is unbearably loud.
